com.unity.netcode.gameobjects/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -385,7 +385,7 @@ internal void ChangeOwnership(NetworkObject networkObject, ulong clientId)
         /// <summary>
         /// Should only run on the client
         /// </summary>
-        internal NetworkObject CreateLocalNetworkObject(bool isSceneObject, uint globalObjectIdHash, ulong ownerClientId, ulong? parentNetworkId, Vector3? position, Quaternion? rotation, bool isReparented = false)
+        internal NetworkObject CreateLocalNetworkObject(bool isSceneObject, uint globalObjectIdHash, ulong ownerClientId, ulong? parentNetworkId, int? networkSceneHandle, Vector3? position, Quaternion? rotation, bool isReparented = false)
```

Comment:
Small nit, but I wonder what the pros/cons are of this vs just taking an int and assigning `NO_SCENE_HANDLE = -1`. The latter might perform a little better... but probably immeasurably so. So I'm just wondering which one is more elegant.

Reply:
-1 is a valid scene handle...

Reply:
Ah. Well then. Nevermind me. *disappears*